index.js

```diff
@@ -91,6 +92,21 @@ module.exports = function(input, inputMap) {
 			});
 			return;
 		}
+
+		if (params.includeModulePaths) {
+			// Ensure module paths include full path from current working directory
+			var pwd = process.cwd();
+			if (context.substring(0, pwd.length) === pwd) {
```

Comment:
Oh, I see what you're saying. I didn't extract it out because I don't see it being used anywhere else. IMO, anonymous functions in that case. But I can extract it out if need be.